cpu/pic32/Makefile.pic32

```diff
@@ -43,6 +46,13 @@ endif
 
 ### Compilation rules
 
+CUSTOM_RULE_LINK=1
+
+%.$(TARGET): %.co $(PROJECT_OBJECTFILES) $(PROJECT_LIBRARIES) contiki-$(TARGET).a
```

Comment:
It'd be good to get some clarification regarding what it does & if we need all of those bits to do the rule.